README.md

```diff
@@ -99,6 +101,21 @@ pip3 install ffmpeg-normalize
 
 Or download this repository, then run `pip3 install .`.
 
+## Docker Build
+Download this repository and run
+
+```
+docker build -t ffmpeg-normalize .
+```
+
+Run using Windows Powershell or Linux:
+```
+docker run  -v "$(pwd):/tmp" -it docker.io/library/ffmpeg-normalize /bin/sh
+```
+This will mount your current folder to the /tmp directory inside the container
+
+Note: The container will run in interactive mode.
```

Comment:
Perhaps add a simple example of which command to run from within the container.

Reply:
The commands are exactly the same as if you're running this from your PC.
Once you run this command:
docker run  -v "$(pwd):/tmp" -it docker.io/library/ffmpeg-normalize /bin/sh
You're literally running sh inside of the container, and you're able to run ffmeg-normalize commands directly.

I've give a simple example :)